Sources/SwiftGen/Commander/TemplatesCommands.swift

```diff
@@ -36,6 +36,40 @@ enum TemplatesCLI {
     }
   }
 
+  static let doc = command(
+    Argument<String?>("subcommand", description: "the name of the subcommand for the template, like `strings`"),
+    Argument<String?>("template", description: "the name of the template to find, like `swift5` or `flat-swift5`")
+  ) { subcommand, template in
+    var path = "templates/"
+    if let subcommand = subcommand {
+      // If we have a subcommand argument, ensure that is one of the allowed ones
+      guard let parserCLI = ParserCLI.command(named: subcommand) else {
```

Comment:
(thinking out loud): I wonder if `Commander` would allow us to write `Argument<ParserCLI?>` instead of `Argument<String?>` if we make `ParserCLI` conform to `ArgumentConvertible`… and if so maybe there are some other places and commands where we could then use this trick? 🤔  Meh maybe not worth it after all 🤷‍♂️